xds/internal/balancer/outlierdetection/config.go

```diff
@@ -154,8 +153,10 @@ type LBConfig struct {
 	ChildPolicy *internalserviceconfig.BalancerConfig `json:"childPolicy,omitempty"`
 }
 
-// Equal returns whether the LBConfig is the same with the parameter.
-func (lbc *LBConfig) Equal(lbc2 *LBConfig) bool {
+// EqualODOnly returns whether the LBConfig is same with the parameter outside
+// of the child policy, only comparing the Outlier Detection specific
+// configuration.
+func (lbc *LBConfig) EqualODOnly(lbc2 *LBConfig) bool {
```

Comment:
Nit:

`EqualIgnoringChildPolicy`?

"ODOnly" seems less informative to me since we're comparing OD LB policy structs in the OD package.

Reply:
Switched.